README.md

```diff
@@ -360,37 +354,50 @@ And what is this ownership worth?
 
 ### How does the value of equity change?
 
-We’ve now seen how complex it is to assess the value of equity in private companies.
-Well, it doesn’t get any easier, because if a company is giving you equity, the intention
-is to grow the company, and so the value of your equity is always changing over time.
+We’ve now seen how hard it is to evaluate the value of equity in private companies,
```

Comment:
Surprisingly common stylistic issue so I'll mention it here (though Josh I can take care of these changes after merge if need, just noting stuff): verbs should never contain the object they act upon, e.g. "evaluate the value." Try "determine" or "establish"